parley/src/layout/mod.rs

```diff
@@ -84,13 +84,13 @@ impl<B: Brush> Layout<B> {
     /// Returns the minimum content width of the layout. This is the width of the layout if _all_
     /// soft line-breaking opportunities are taken.
     pub fn min_content_width(&self) -> f32 {
-        self.data.min_content_width
+        self.data.content_widths().0
     }
 
     /// Returns the maximum content width of the layout. This is the width of the layout if _no_
     /// soft line-breaking opportunities are taken.
     pub fn max_content_width(&self) -> f32 {
-        self.data.max_content_width
+        self.data.content_widths().1
     }
```

Comment:
A method that returns both could also be useful. Would work well in combination with a `ContentWidths` struct proposed above.

Reply:
I meant in addition. These methods are cheap for us to add. And I think it's nice for the user to have the option to pick whatever they need.